src/components/MUILinkWithTracking/MUILinkWithTracking.stories.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import type { Meta, StoryObj } from '@storybook/react';
+import { MUILinkWithTracking } from '.';
+import { BrowserRouter, Route, Routes } from 'react-router-dom';
+
+const meta = {
+	title: 'Other/MUILinkWithTracking',
+	component: MUILinkWithTracking,
+	tags: ['autodocs'],
+} satisfies Meta<typeof MUILinkWithTracking>;
+
+export default meta;
+type Story = StoryObj<typeof meta>;
+
+export const Default: Story = {
+	render: (args) => (
+		<BrowserRouter>
+			<Routes>
+				<Route
+					path="*"
+					element={
+						<React.Fragment>
+							<MUILinkWithTracking {...args} />
+						</React.Fragment>
+					}
+				/>
+			</Routes>
+		</BrowserRouter>
+	),
+	args: {
+		href: '/',
+		children: 'Material Link with analytics',
```

Comment:
we probably need to add an onClick to avoid redirects or page reloads:

```suggestion
		children: 'Material Link with analytics',
	        onClick: (e) => {
			e.preventDefault();
		},
```